services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -3998,6 +4008,94 @@ func TestSendRegularlyDatadogMetrics(t *testing.T) {
 	}
 }
 
+type MockClient struct {
+	events []*statsd.Event
```

Comment:
You can remove most stub implementations by adding `statsd.ClientInterface` here.

Reply:
done